lib/client/mfa/cli.go

```diff
@@ -87,6 +72,12 @@ func NewCLIPromptV2(cfg *CLIPromptConfig) *CLIPrompt {
 	}
 }
 
+// NewCLIPromptV2 returns a new CLI mfa prompt with the given config.
+// TODO(Joerger): remove once /e is no longer dependent on this.
+func NewCLIPromptV2(cfg *CLIPromptConfig) *CLIPrompt {
```

Comment:
When the e/ update lands we can remove this, right?

Reply:
Yeah, so 2 more follow up PRs after this 😅